tests/e2e/tests_runner.sh

```diff
@@ -112,6 +100,49 @@ main() {
 		"agent_image.bats" \
 		"agent_image_encrypted.bats" \
 		"${script_dir}/operator_tests.bats"
+
+}
+
+# Tests for CC with QEMU on SEV HW
+run_kata_qemu_sev_tests() {
+	local tests_passing="Test SEV unencrypted container launch success"
+	tests_passing+="|Test SEV encrypted container launch failure with INVALID measurement"
+	tests_passing+="|Test SEV encrypted container launch success with NO measurement"
+	tests_passing+="|Test SEV encrypted container launch success with VALID measurement"
+	bats -f "$tests_passing" \
+		"sev.bats"
+
+}
+
+main() {
+
+	parse_args $@
+
+	clone_kata_tests
+
+	cd "${tests_repo_dir}/integration/kubernetes/confidential"
+
+	# Test scripts rely on kata-runtime so it should be reacheable on PATH.
+	# Re-export PATH is error prone as some calls to kata-runtime use sudo,
+	# so let's create a symlink.
+	ln -sf /opt/confidential-containers/bin/kata-runtime \
+		/usr/local/bin/kata-runtime
+
+	# Run tests.
+	case $runtimeclass in
+		kata-qemu)
+			echo "INFO: Running kata-qemu tests"
+			run_kata_qemu_tests
+			;;
+		kata-qemu-sev)
+			echo "INFO: Running kata-qemu-sev tests"
+			run_kata_qemu_sev_tests
+			;;
+		*)
```

Comment:
@arronwy we will need to add an entry for TDX here. IIUC we will be running the non-TEE tests (i.e. same as in run_kata_qemu_tests()) for TDX but tweaking the runtimeclass. Is that what you want?

Moreover, if we merge it as is it will break your tests you have made manually. Is it okay to merge it anyway then we open a follow up PR to add the TDX tests in?

Reply:
> @arronwy we will need to add an entry for TDX here. IIUC we will be running the non-TEE tests (i.e. same as in run_kata_qemu_tests()) for TDX but tweaking the runtimeclass. Is that what you want?

Yes, non-TEE tests can be the common test cases for all the runtimeclasses, we can set correct  `RUNTIMECLASS`  afterthis PR: https://github.com/kata-containers/tests/pull/5131 is merged.

> Moreover, if we merge it as is it will break your tests you have made manually. Is it okay to merge it anyway then we open a follow up PR to add the TDX tests in?

It's ok for me to merge this PR and a follow up PR for other runtimeclasses with default non-TEE tests.

Reply:
> > @arronwy we will need to add an entry for TDX here. IIUC we will be running the non-TEE tests (i.e. same as in run_kata_qemu_tests()) for TDX but tweaking the runtimeclass. Is that what you want?
> 
> Yes, non-TEE tests can be the common test cases for all the runtimeclasses, we can set correct `RUNTIMECLASS` afterthis PR: [kata-containers/tests#5131](https://github.com/kata-containers/tests/pull/5131) is merged.
>

Notice that some tests still write to kata's configuration.toml and I don't know how to avoid that.
 
> > Moreover, if we merge it as is it will break your tests you have made manually. Is it okay to merge it anyway then we open a follow up PR to add the TDX tests in?
> 
> It's ok for me to merge this PR and a follow up PR for other runtimeclasses with default non-TEE tests.

Cool. so I will merge it.